### ISSUE TITLE
documentation included in ontology releases

### ISSUE THREAD

**Issue by stuckyb** [stuckyb/ontobuilder#79]
It would probably be useful to include ontology documentation as part of an ontology release.  That would freeze the ontology, import modules, and documentation together as a unit, which might be useful.  Anyone else have thoughts on this?  @ramonawalls, based on your experience, would this be a good practice?  I realize that I've not yet shown the rest of you OntoPilot's new ontology documentation facilities; I hope to have that polished up enough soon.

**Comment by ramonawalls** [comment]
Yes, Brian, I agree that it is important to have some good ontology documentation to go along with the release and paper. Not everyone does it, but it will really help users, and probably reviewers too. I'm happy to help with the documentation, after vacation!
